qip-0009.md

```diff
@@ -0,0 +1,529 @@
+ ```
+  QIP: 9
+  Layer: Consensus (hard fork)
+  Title: Interlinks in Quai
+  Author: gameofpointers <gameofpointers+qip@quai.org>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/quainetwork/qips/wiki/Comments:QIP-0009
+  Status: Draft
+  Type: Standards Track
+  Created: 2023-11-16
+  License: BSD-2-Clause
+ ```
+
+## Abstract
+
+This QIP defines a block interlink structure to be used in Quai as described in
+[1] with few modifications.
+
+## Motivation
+
+To Prove/Verify the Proof Of Work done on Quai since the genesis using a
+non-interactive proof of proof of work (NiPoPoW). Also verify a transaction inclusion in a Block at a depth for which the light node(SPV client) doesn't have the state to verify. This requires using only
```

Comment:
You have 80c line breaks in some places, but not others. Should we try to stick to 80c lines everywhere in QIPs?

Reply:
Yeah, will keep it consistent with 80c